File: test/dovetail/basic_test.rb
require 'test_helper'

describe 'dovetail-basic' do

  it 'gets the root path' do
    resp = Excon.get(CONFIG.DOVETAIL_HOST)
    resp.status.must_equal 200
    resp.headers['content-type'].must_include 'text/plain'
    resp.body.to_s.must_equal "Dovetail. You know, for podcasts."
  end

end

File: test/dovetail/basic.test.js
const fetch = require('node-fetch');

describe('dovetail - basic', () => {
  it('has a root doc', async () => {
    const res = await fetch(DOVETAIL_HOST);
    const text = await res.text();

    expect(res.status).toEqual(200);
    expect(res.headers.get('content-type')).toMatch(/text\/plain/);
    expect(text).toContain('You know, for podcasts');
  });
});
